1.0: Add tech preview disclaimer (#528)
SUSE-Cloud/socok8s: doc/source/index.rst

Welcome to SUSE Containerized OpenStack Technology Preview
==========================================================

The socok8s project automates SUSE Containerized OpenStack provisioning and
lifecycle management on SUSE Container as a Service Platform (CaaSP) and
SUSE Enterprise Storage (SES), using Airship, OpenStack Helm, shell scripts,
and Ansible playbooks.

As a technology preview, the included features are not supported. They may be
functionally incomplete, unstable or in other ways not suitable for production

> use. They are mainly included for customer convenience and give customers a

Reviewer: s,customers,users,

Author: done

chance to test new technologies within an enterprise environment.

This release deploys the following set of core OpenStack services:

  * Keystone
  * Glance
  * Cinder
  * Nova
  * Neutron
  * Heat
  * Horizon

> Whether it will be moved to a fully supported product later depends on customer

Reviewer: s,on customer,on your,

Author: done

and market feedback. A technology preview does not automatically result in

> support at a later point in time. It could be dropped at any time and SUSE is not

Reviewer: ```suggestion
support at a later point in time. It is provided under the terms of the Apache-2.0 license and SUSE is not
```

> committed to providing a technology preview later in the product cycle.

Reviewer: ```suggestion
comitting to providing a technology preview later in the product cycle.
```

Author: I think the right grammar is "committed"

Please, give your SUSE representative feedback, including your experience and

> use case.

Reviewer: ```suggestion
relevance to your use case.
```
